### PR TITLE
Include more guidance on embassy-rs installation

### PR DESCRIPTION
**Pull request by UdrAlberto**
Included fix for missing dependency error

### Pull Request Overview

This pull request adds more information on the Linux installation process

### Testing Strategy

This pull request was tested by Alberto Udrea

### TODO or Help Wanted

There might be a better way to include the shell block after the note

### Build

- [x] Ran `npm build`.

### Author

Signed-off-by: Alberto Udrea <albertoudrea4@gmail.com>

### CODE REVIEW

File: docs/tutorials/embassy/index.md [hunk 1]
```diff
@@ -66,7 +66,10 @@ Options:
   -s, --serial   Connect to serial after deploy
   -h, --help     Print help
 ```
-
+:::info Before installing elf2uf2-rs, you might need an additional package, called libudev. You can get it by running the following in your terminal. :::
+```shell
+sudo apt-get install libudev-dev
+```
```
Comment (alexandruradovici) [inline docs/tutorials/embassy/index.md:72]: This is only valid for Linux, please move it to the Linux part.

File: docs/tutorials/embassy/index.md [hunk 2]
```diff
@@ -66,7 +66,10 @@ Options:
   -s, --serial   Connect to serial after deploy
   -h, --help     Print help
 ```
-
+:::info Before installing elf2uf2-rs, you might need an additional package, called libudev. You can get it by running the following in your terminal. :::
```
Comment (alexandruradovici) [inline docs/tutorials/embassy/index.md:69]:
```suggestion
:::info Before installing elf2uf2-rs, you might need an additional package, called `libudev`. You can get it by running the following in your terminal. :::
```

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

File: docs/tutorials/embassy/index.md (hunk 1 above)
Comment (UdrAlberto) [reply docs/tutorials/embassy/index.md]: Moved it, sorry for the inconvenience

File: docs/tutorials/embassy/index.md (hunk 2 above)
Comment (UdrAlberto) [reply docs/tutorials/embassy/index.md]: Also added special formatting to `libudev`

File: docs/tutorials/embassy/index.md [hunk 3]
```diff
@@ -24,6 +24,11 @@ curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
 
 This downloads and runs `rustup-init.sh`, which in turn downloads and runs the correct version of the `rustup-init` executable for your platform.
 
+:::info Before installing elf2uf2-rs, you might need an additional package, called `libudev`. You can get it by running the following in your terminal. :::
+```shell
+sudo apt-get install libudev-dev
+```
```
Comment (alexandruradovici) [inline docs/tutorials/embassy/index.md:30]:
```suggestion
:::info 

Before installing elf2uf2-rs, you might need an additional package, called `libudev`. You can get it by running the following in your terminal. 
```shell
sudo apt-get install libudev-dev
```

:::
```

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

**Review by alexandruradovici — APPROVED** [review]

**Review by DanutAldea — APPROVED** [review]

**Comment by UdrAlberto** [comment]
Just noticed that we might've missed a `:::` because the Windows section is now included in the Linux note.
image

**Comment by alexandruradovici** [comment]
Please send a PR ti fix this.

**Pull request merged**